content/en/developers/authorization/oauth2_endpoints.md

```diff
@@ -0,0 +1,234 @@
+kind: documentation
+description: Learn how to use the OAuth2 authorization endpoints.
+further_reading:
+- link: "/developers/authorization/"
+  tag: "Documentation"
+  text: "Learn about OAuth2 Authorization"
+---
+{{< beta-callout btn_hidden="true" >}}
+  The Datadog Developer Platform is currently in beta. If you don't have access, contact apps@datadoghq.com.
+{{< /beta-callout >}} 
+
+## Overview
+
+Applications using protected Datadog resources must be authorized by a user before they can access Datadog APIs on the user’s behalf. These endpoints direct the application through the authorization code grant flow. 
+
+{{< tabs >}}
+{{% tab "Authorization Endpoints" %}}
+
+## Request authorization from a user
+
+### `GET /oauth2/v1/authorize`
+
+#### Overview
+
+To start the authorization code grant flow, an application makes a `GET` request to Datadog’s authorization endpoint. This redirects a user to Datadog’s authorization-grant flow and renders a consent page displaying the list of scopes requested by your application and a prompt for the user to authorize access. 
+
+#### Request 
+In the authorization request, the application constructs the redirect URI by adding the following parameters to the query component of the URI using the `application/x-www-form-urlencoded` format: 
+
+| URL Parameter                               | Description                                                                                               |
+|---------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `redirect_uri`                                | Your application’s redirection endpoint after a user grants or denies access.                              |
+| `client_id`                                   | The Client ID of your OAuth2 client.                                                                       |
+| `response_type`                               | The response type must be code for this grant flow.                                                        |
+| `code_challenge`  (if PKCE is enabled)        | A transformation of `code_verifier`. Datadog recommends using `SHA-256` to compute the code challenge.     |
+| `code_challenge_method`  (if PKCE is enabled) | The method used to compute the code challenge. `SHA-256`, or `S256`, is supported.  |
+
+#### Example Request
+
+To render Datadog’s consent page, redirect users to the endpoint with the specified parameters: 
+```
+https://app.datadoghq.com/oauth2/v1/authorize?redirect_uri=http://localhost:500/oauth_redirect&client_id=abcdefghijklmnopqrstuvwxyz_123456789&response_type=code&code_challenge=12345&code_challenge_method=S256
+```
+
+#### Success Response
+
+If a user successfully grants the access request, your application [obtains an authorization code](#Obtain-an-authorization-code) and redirects the user to the redirect URI with the authorization `code` in the query component.
```

Comment:
I think we should also add that the `site` query parameter is also included with the redirect URI, which is used to extract the user's datacenter

Reply:
Good callout, I've mentioned this here and in the overview in my upcoming commit